src/applications/ivc-champva/10-10D/components/Sponsor/sponsorFileUploads.js

```diff
@@ -147,7 +147,10 @@ export function uploadWithInfoComponent(fileList, category, resources) {
           </>
         ),
       },
-      ...fileUploadBlurb,
+      ...fileUploadBlurbCustom(
+        undefined,
+        'If you don’t want to upload your supporting files now, you’ll have the option to upload again at the end of the application.',
+      ),
```

Comment:
Switching to use the custom blurb component that supports arbitrary follow-up text.